### DISCUSSION TITLE
Build not minified in production?

### DISCUSSION THREAD

**Discussion by lsbyerley** [epicweb-dev/epic-stack#738] [Q&A]
I've noticed the build isn't minified when deployed to fly in the production environment. I need to look more into vite but I don't see any minification config for js, or maybe I'm missing it. Is there any reason why it's not by default?

**Comment by kentcdodds** [comment]
I just checked a couple assets from the deployed version of the Epic Stack and they're definitely minified: https://www.epicstack.dev/assets/entry.client-BKTMPNy1.js

What made you think they're not?

**Reply by lsbyerley** [reply]
My mistake.. when looking at the file in devtools, it doesn't appear to be because of sourcemaps, but looks fine loaded in the browser.
